src/util/initial-config.yaml

```diff
@@ -114,6 +114,7 @@ full_node:
 
   # Run multiple nodes with different databases by changing the database_path
   database_path: db/blockchain_v18.db
+  peer_table_path: db/peer_table.db
```

Comment:
Why use a new DB? We can use the same DB but different table (I think it's simpler)/

Reply:
Bitcoin uses a separate peer db, and I kind of lean towards that too... sometimes people want to remove just the blockchain or just the peer db, and this makes it easier. Not sure what the advantage of having them all in one file is... disk savings would be minimal, and we don't have an especially large of files in this directory.